src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxDocumentConverter.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using Corax;
+using Raven.Client;
+using Raven.Server.Json;
+using Raven.Server.Utils;
+using Sparrow;
+using Sparrow.Extensions;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Sparrow.Server;
+using Sparrow.Threading;
+using Voron;
+
+namespace Raven.Server.Documents.Indexes.Persistence.Corax
+{
+    public sealed class CoraxDocumentConverter : ConverterBase, IDisposable
```

Comment:
Wouldn't make sense to make `ConverterBase` disposable and have `public abstract void Dispose();` there? Currently if you assign `CoraxDocumentConverter` instance to `ConverterBase` field somewhere then you might not notice that is needs to be disposed.

Reply:
Make sense, done.